packages/plugin-window-onerror/test/onerror.test.js

```diff
@@ -53,7 +46,7 @@ describe('plugin: window onerror', () => {
     //   const client = new Client(VALID_NOTIFIER)
     //   const payloads = []
     //   client.setOptions({ apiKey: 'API_KEY_YEAH' })
-    //   client.configure()
+    //   client._configure()
```

Comment:
It's only a comment but I think this should be unchanged. Maybe the all commented code needs updating or just removing? Same for other places in this file.

Reply:
We have a backlog ticket to convert all these commented out unit tests into maze runner tests.

I'll undo the change.